packages/editor/src/utils/index.js

```diff
@@ -4,3 +4,4 @@
 import mediaUpload from './media-upload';
 
 export { mediaUpload };
+export { filterURLForDisplay } from './url';
```

Comment:
Should we move it to `@wordpress/url` instead?

Reply:
Sounds good. I just don't want too many code changes in this PR :D